lib/widgets-filters.php

```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * Add filters for widgets.
+ *
+ * @package gutenberg
+ */
+
+add_filter( 'wp_get_widget_key', 'gutenberg_get_widget_key' );
```

Comment:
@spacedmonkey Why a new file? Why not add to `lib/widgets.php`?

Reply:
`lib/widgets.php` is loaded before `widgets-api.php`. 

In my testing moving this filter to `widgets.php` made a filter. 

I was also thinking of moving existing widget filters / actions to this file.